pages/cart.tsx

```diff
@@ -96,11 +154,47 @@ const Cart = () => {
                               >
                                 <span>Remove</span>
                               </button>
+                              {quantityErrors && (
+                                <div>
+                                  {quantityErrors.map((error: any) => (
+                                    <span
+                                      className="text-red-500 text-sm font-medium"
+                                      key={error.field}
+                                    >
+                                      {error.message}
+                                    </span>
+                                  ))}
+                                </div>
+                              )}
+                            </div>
+                            <div className="flex justify-items-end space-x-4">
+                              <input
+                                type="number"
+                                className={clsx(
+                                  "h-8 w-16 block border-gray-300 rounded-md shadow-sm sm:text-sm",
+                                  quantityErrors && "border-red-500"
+                                )}
+                                value={
```

Comment:
When we separate component for lines, managing value will be easier too (no longer need to manage list for all lines)